frontend/src/components/FlashMessage.js

```diff
@@ -0,0 +1,22 @@
+
+
+export const FlashMessage = ({ messageState, returnError }) => {
```

Comment:
I don't understand how this component flashes a message? It doesn't seem to be able to flash?

Reply:
it is timed out